python/lsst/meas/algorithms/defects.py

```diff
@@ -151,6 +151,25 @@ def __eq__(self, other):
     def __str__(self):
         return "Defects(" + ",".join(str(d.getBBox()) for d in self) + ")"
 
+    def normalize(self, region):
```

Comment:
Why is this taking a mandatory argument? If you want the the normalized defects to refer to a subset then an optional argument here is fine, but by default you should work this region out yourself.  Your defects object has a bunch of regions in it -- work out the minimum and maximum bounds from all those defect regions and use them.

Reply:
I put a simple unit test but in "cp_pipe/tests/test_defects.py". I'll move it to somewhere in "meas_algorithms/tests/".

Reply:
Ok. I do think that at least one of the tests should normalize two distinct sets of Defects and ensure they compare to the same thing.